### PR TITLE
Add package to cloudsmith for testing and internal use

### PR DESCRIPTION
**Pull request by doc-E-brown**
- [ ] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [ ] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: geo/Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,10 @@ voronoi = ["delaunay"]
 earcutr = { version = "0.4.2", optional = true }
 spade = { version = "2.2.0", optional = true }
 float_next_after = "1.0.0"
-geo-types = { version = "0.7.9", features = ["approx", "use-rstar_0_11"] }
+geo-types = { version = "0.7.11", features = [
```
Comment (jayvdb) [inline geo/Cargo.toml]: Shouldnt we have published geo-types to cloudsmith as something like `0.7.12-alpha1` ?

Comment (doc-E-brown) [reply geo/Cargo.toml]: I haven't made any changes to `geo-types` I just needed to publish it into cloudsmith to package `geo` for cloudsmith.  But I do need to make this change for geo

Comment (doc-E-brown) [reply geo/Cargo.toml]: This change just bumps geo-types to the latest version

Comment (doc-E-brown) [reply geo/Cargo.toml]: This has been addressed by rebasing against upstream

**Review by jayvdb — APPROVED** [review]

**Pull request merged**